worker/sshserver/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+//
```

Comment:
If I'm not mistaken, you need to remove the comment in this line otherwise the package godoc.